images/README.md

```diff
@@ -10,3 +10,22 @@ To add additional applications into the images, open a PR with changes. Follow t
 * Always build from a source to ensure compiler vulnerabilities do not affect the resulting binary
 * Link the binary to a specific version so that it's easier to update when necessary 
 * Build binaries in a separate stage, then copy the resulting binary into the final image to ensure images are small and contain the least number of layers
+
+## Writing image tests
+
+to write simple smoke tests with your image, add an **executable** file called `test.sh`.
```

Comment:
```suggestion
To write simple smoke tests with your image, add an executable file called `test.sh`.
```

Reply:
I don't understand. I've purposely set the `executable` word in bold to highlight that this script must executable.

Reply:
Please, see what we use bold font for in the [guidelines](https://github.com/kyma-project/community/blob/main/docs/guidelines/content-guidelines/03-formatting.md#use-bold-font-for-these-items).